src/DotNetBridge/Entrypoints.cs

```diff
@@ -10,6 +10,11 @@
 
 [assembly: LoadableClass(typeof(void), typeof(DotNetBridgeEntrypoints), null, typeof(SignatureEntryPointModule), "DotNetBridgeEntrypoints")]
 
+[assembly: LoadableClass(VariableColumnTransform.Summary, typeof(VariableColumnTransform), null, typeof(SignatureLoadDataTransform),
+    "", VariableColumnTransform.LoaderSignature)]
+
+[assembly: LoadableClass(typeof(void), typeof(VariableColumnTransform), null, typeof(SignatureEntryPointModule), "VariableColumnTransform")]
```

Comment:
>VariableColumnTransform [](start = 114, length = 23)

should be DotNetBridgeEntrypoints. its a class name where entrypoint is from

Reply:
The line has been removed in commit 0b3c993e650daf2e153afdfb0b227a0c1f6c0335.